prestoadmin/server.py

```diff
@@ -82,11 +85,218 @@ def new_sysnode_processor(node_info_rows):
 CONNECTOR_INFO_SQL = 'select catalog_name from system.metadata.catalogs'
 _LOGGER = logging.getLogger(__name__)
 
+DOWNLOAD_DIRECTORY = main_dir
```

Comment:
This seems like a bad choice for anybody running with a small root filesystem. /tmp is probably a better choice. I'd argue that anybody downloading the RPM automatically probably isn't interested in keeping it around forever.

We should probably print a message saying where the RPM gets downloaded to.